tools/st2ctl

```diff
@@ -231,6 +231,13 @@ function getpids(){
       done
     else
       echo "${COM} is not running"
+      if [ "$COM" == "st2sensorcontainer" ] || [ "$COM" == "sensor_container" ]
+      then
+        echo "Sensor container did not detect valid sensors to run."
+        echo "Please install sensors or check sensors configuration for existing ones."
+        echo "See http://docs.stackstorm.com/sensors.html."
+        echo "Use 'st2ctl restart-component st2sensorcontainer' to restart after you fix the problems."
```

Comment:
Do we need to have `base_pack` actually passed here? It's a first level construct of the config... would it be sufficient enough to just actually say `base_pack` as opposed to filling in?